src/components/ArticlePage.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { getArticle } from '../utils/api-requests'
+
+export default function articlePage(){
+
+    const {article_id} = useParams()
+    const [articleInfo, setArticleInfo] = useState([])
+    const [articleTimeStamp, setArticleTimeStamp] = useState([])
+
+    useEffect(() => {
+        getArticle(article_id)
+            .then(({ article }) => {
+                setArticleInfo(article[0]) 
```

Comment:
You could handle the article[0] in the api file so that is is just article when you get to this point